config/configuration.toml.in

```diff
@@ -1,6 +1,6 @@
 # XXX: Warning: this file is auto-generated from file "@CONFIG_IN@".
 
-[hypervisor.qemu-lite]
+[hypervisor.q35]
```

Comment:
I will modify the tests to build Q35 and install it. BTW, there is a password when cloning the repo `git clone ssh://docker@vmm-docker1.bj.intel.com/intel/git/qemu.git, how am I supposed to get this done from the script ?